_includes/manuals/2.0/1.2_release_notes.md

```diff
@@ -421,6 +421,7 @@ External authentication sources can now be assigned to taxonomies using a new UI
 *A full list of changes in 2.0.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=1132)*
 
 ### Deprecations
+* Deprecate Rackspace Compute Resource ([#29285](https://projects.theforeman.org/issues/29285)) 
```

Comment:
Please add a bit more information beyond just the RM title, and maybe link to the discourse post?

Reply:
thanks @tbrisker I've added the link, but I'm not sure which more information is needed here.